util/argo/audit_logger.go

```diff
@@ -130,6 +131,16 @@ func (l *AuditLogger) LogAppProjEvent(proj *v1alpha1.AppProject, info EventInfo,
 	l.logEvent(objectMeta, v1alpha1.AppProjectSchemaGroupVersionKind, info, message, nil)
 }
 
+func (l *AuditLogger) LogAppSetEvent(appset *appset.ApplicationSet, info EventInfo, message string) {
```

Comment:
Please remove this method, since it is not yet used anywhere. Let's introduce it when we start building application set API

Reply:
yeah sure